### PR TITLE
Set a Claro theme for Backend

### LINKED ISSUE #23: Use Claro as the admin theme
## Description
In the future we will create our own theme, but for now let's use Olivero as the default as a starter
## Tech notes
Install and set Claro theme as the admin theme.
Uninstall Seven theme.

## Success Criteria

- [ ] Claro theme is the admin theme
- [ ] Seven theme is uninstalled.

## Test Instructions

What: 
When: 
Where: 
How:

### PR DESCRIPTION
**Pull request by Luis93Hub**
[Use Claro as the admin theme #23]
(https://github.com/edysmp/stand-blog/issues/23)    
https://github.com/edysmp/stand-blog/issues/23  Issue

Fixes #23 

Install Claro Theme for Admin theme to Backend Dev.

image

### CODE REVIEW

File: config/sync/core.extension.yml [hunk 1]
```diff
@@ -58,4 +58,5 @@ module:
 theme:
   bartik: 0
   seven: 0
```
Comment (edysmp) [inline config/sync/core.extension.yml:60]: let's uninstall the `seven` theme too.

**Comment by Luis93Hub** [comment]
The seven Theme was deleted, and did commit and push.

**Comment by edysmp** [comment]
> The seven Theme was deleted, and did commit and push.

Is this PR only finxing #23? or adding Olivero too?

**Comment by edysmp** [comment]
closing in favor of https://github.com/edysmp/stand-blog/pull/30

**Pull request closed without merging**